pages/levels/1.vue

```diff
@@ -104,13 +129,14 @@ export default {
 
     <div class="flex flex-1 flex-col items-stretch gap-8">
       <div
-        class="prose prose-sm flex-1 overflow-y-auto rounded-2xl border-2 bg-bg-light p-8 text-base dark:prose-invert dark:bg-[#293e74]">
+        class="prose prose-lg flex-1 overflow-y-auto rounded-2xl border-2 p-8 dark:prose-invert border-blue-4 dark:bg-blue-3-dark bg-blue-3-light shadow-content-box-drop-shadow">
         <VueMarkdown :source="md" />
       </div>
 
       <NuxtLink to="2">
-        <div class="rounded-2xl border-2 p-1 flex justify-center" :class="{
-              'bg-green-400': correct,
+        <div class="rounded-2xl p-1 flex justify-center  dark:text-bg-blue-1-dark" :class="{
```

Comment:
hier ist ein space zu viel, bitte autoformatieren mit prettier (entweder in IDE) oder mit `npm run prettier`